lib/logger.js

```diff
@@ -1,17 +1,38 @@
 'use strict';
 // Copyright (C) 2020 Splunk, Inc. All rights reserved.
 
-var winston = require('winston');
+var levels = {
+  error: 1,
+  info: 2,
+  warn: 3,
+  debug: 4
+};
 
-var logger = winston.createLogger({
-  level: process.env.SFX_CLIENT_LOG_LEVEL || 'info',
-  format: winston.format.combine(
-      winston.format.splat(),
-      winston.format.simple()
-  ),
-  transports: [
-    new winston.transports.Console()
-  ]
-});
+var methods = {
+  error: console.error,
+  info: console.info,
+  warn: console.warn,
+  debug: console.debug
+};
 
-module.exports = logger;
+var defaultLevel = levels[process.env.SFX_CLIENT_LOG_LEVEL || 'info'];
+
+function createLogger(level) {
+  return function () {
+    if (defaultLevel && levels[level] > defaultLevel) {
```

Comment:
Since `defaultLevel` is module-local and cannot be changed, you could do that check outside of the logger function(called on every log) and inside the `createLogger`(called once) - just return noop function.